mongo_connector/connector.py

```diff
@@ -704,19 +713,53 @@ def apply_namespaces(option, cli_values):
             raise errors.InvalidConfiguration(
                 "Namespace set should not contain any duplicates.")
 
+        ex_ns_set = option.value['exclude']
+        if len(ex_ns_set) != len(set(ex_ns_set)):
+            raise errors.InvalidConfiguration(
+                "Exclude namespace set should not contain any duplicates.")
+
+        # not allow to exist both 'include' and 'exclude'
+        if ns_set and ex_ns_set:
+            raise errors.InvalidConfiguration(
+                "Include and exclude namespace sets"
+                " are not allowed to exist both.")
```

Comment:
Change to "Cannot use both namespace 'include' (--namespace-set) and 'exclude' (--exclude-namespace-set)."